mongo/collection.go

```diff
@@ -1632,6 +1969,52 @@ func (coll *Collection) FindOneAndReplace(ctx context.Context, filter interface{
 	return coll.findAndModify(ctx, op)
 }
 
+// mergeFindOneAndUpdateOptions combines the given FindOneAndUpdateOptions instances into a single
+// FindOneAndUpdateOptions in a last-one-wins fashion.
```

Comment:
Optional: see similar for explanation

```suggestion
// FindOneAndUpdateOptions in a last-property-wins fashion.
```